src/transformers/models/mistral/modeling_mistral.py

```diff
@@ -729,12 +715,9 @@ def forward(
         past_key_value: Optional[Tuple[torch.Tensor]] = None,
         output_attentions: Optional[bool] = False,
         use_cache: Optional[bool] = False,
+        cache_position: Optional[torch.LongTensor] = None,
         **kwargs,
```

Comment:
```suggestion
```

Reply:
Yeah, it' that the llama on main branch still have `padding_mask` and `kwargs` logic

Reply:
maybe I can modify that in llama and get a copied from

Reply:
Yep!